subsys/bluetooth/controller/Kconfig

```diff
@@ -462,7 +462,6 @@ config BT_CTLR_ADV_SET
 config BT_CTLR_ADV_DATA_LEN_MAX
 	int "Maximum Advertising Data Length"
 	range 31 1650
-	default 251
```

Comment:
Hmm, I wonder how many, if any, samples that will stop working by defaulting to 31 instead of 251.

Reply:
Only the periodic advertising and direction finding connectionless tx samples enable the advertising extensions feature in the Controller, 31 octets are sufficient for these samples. @ppryga could confirm on the direction finding samples...

Reply:
DF samples and the functionality itself does not imply to extend the advertising data length. 31 octets is sufficient for the functionality.